cfgov/unprocessed/css/molecules/translation-links.less

```diff
@@ -58,3 +63,36 @@ html[lang='ar'] ul.m-translation-links {
     }
   }
 }
+
+//fix issue with padding/margin and verticle rule appearing to the left of print icon
+html[lang='ar'] .m-social-media {
+  display: inline-block;
+  direction: rtl;
+
+  &_print {
+    padding-left: 0;
+    border-left: none;
+    padding-right: unit((@grid_gutter-width / 2) / @base-font-size-px, em);
+    border-right: 1px solid @black;
+  }
+
+  & .m-list .m-list_item:last-child {
+    margin-right: 0.6875em;
+  }
+
+  //Position share & print text above social media icons and remove right margin
+  @media screen and (max-width: 420px) {
```

Comment:
We have [media query mixins](https://cfpb.github.io/design-system/development/media-queries) for this use. For this it would be `.respond-to-max( @bp-xs-max, {`, where `@bp-xs-max` comes from https://github.com/cfpb/design-system/blob/main/packages/cfpb-core/src/vars-breakpoints.less#L9

Reply:
media query mixin added to social-media.less